src/index.html

```diff
@@ -12,7 +13,40 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length-31 calendar--start-day-sun">
```

Comment:
It seems you are hardcoding the calendar days. It's not a scalable solution. What if the month has 30 days? Or 28? And what about the starting day? It's better to generate the calendar using JavaScript or a server-side language.